packages/@aws-cdk/integ-tests-alpha/lib/assertions/waiter-state-machine.ts

```diff
@@ -33,6 +34,13 @@ export interface WaiterStateMachineOptions {
    * @default 1 (no backoff)
    */
   readonly backoffRate?: number;
+
+  /**
+   * How long, in days, the log contents will be retained.
+   *
+   * @default - no retention days specified
+   */
+  readonly logRetention?: RetentionDays;
```

Comment:
for the maintainers to decide but do we actually want to set a reasonable default here? maybe 3 days or 1 week.

Reply:
I think that no default is correct here. It's what was also done here https://github.com/aws/aws-cdk/pull/29277#discussion_r1525419143.